css/validation.css

```diff
@@ -2,35 +2,37 @@
    Message Area
    ====================================================================== */
 .message-area {
-  /* Message area to provide feedback used to display messages such as...
-     "Sorry, that email or password is not recognised." or "No comments yet :("
-  */
-  font-size: 1em;
-  color: #31708f;
-  background-color: #d9edf7;
-  border: 1px solid #bce8f1;
-  border-radius: 4px;
-  padding: 10px 20px;
-  margin: 0 0 20px 0;
-  text-align: left;
-  cursor: pointer;
+    /* Message area to provide feedback used to display messages such as...
+       "Sorry, that email or password is not recognised." or "No comments yet :("
+       */
+    font-size: 1em;
+    color: #31708f;
+    background-color: #d9edf7;
+    border: 1px solid #bce8f1;
+    border-radius: 4px;
+    padding: 10px 20px;
+    margin: 0 0 20px 0;
+    text-align: left;
+    cursor: pointer;
+    position: relative;
 }
 
 .message-area > .icon--cross {
-  float: right;
-  width: 10px;
-  margin-top: 5px;
-  content: url("../images/cross.svg");
+    position: absolute;
+    width: 10px;
+    top: calc(50% - 5px);
+    left: calc(98% - 10px);
```

Comment:
There's no svg support in IE 8. We've been using a png fallback when necessary.

Pretty sure Bootstrap uses an icon font to display their icons.
 
I think we eventually came to a good solution for the buttons we did in mla-www-responsive project. Might be able to take something from [there](https://github.com/qcode-software/mla-www-responsive/blob/master/scss/_tools.buttons.scss)